modules/passkey/hardhat.config.ts

```diff
@@ -31,6 +31,18 @@ const customNetwork = CUSTOM_NODE_URL
     }
   : {}
 
+const compilerSettings = {
+  version: '0.8.24',
+  settings: {
+    optimizer: {
+      enabled: true,
+      runs: 10_000_000,
+    },
+    viaIR: true,
+    evmVersion: 'paris',
```

Comment:
paris is pre-push0, right? maybe we don't need this either?

Reply:
We do, so we are BNB compatible.

Also, note that `hardhat` currently defaults to `paris` anyway (at least it did when I experimented with switching this setting off), so this is more to explicitly document the EVM version that is being targetted.

Reply:
it seems like BNB already has push0? https://forum.bnbchain.org/t/bnb-chain-roadmap-mainnet/936

Reply:
This is super recent! It wasn't the case 4 days ago: https://github.com/hats-finance/Safe-0x2909fdefd24a1ced675cb1444918fa766d76bdac/issues/19#issuecomment-2180019350

Then `s/BNB/Linea/g`: I think the point remains the same that `PUSH0` isn't generally well supported across L2s:
```
$curl -s https://rpc.linea.build -H 'content-type: application/json' --data '{"jsonrpc":"2.0","id":0,"method":"et
h_call","params":[{"data":"0x5f"},"latest"]}'
{"jsonrpc":"2.0","id":0,"error":{"code":-32000,"message":"invalid opcode: PUSH0"}}
```

Reply:
interesting, the forum post says it was implemented in `Kepler: 2024/01/23` hardfork